### DISCUSSION TITLE
Small improvements

### DISCUSSION THREAD

**Discussion by farag2** [pratyakshm/WinRice#18] [Ideas]
`C:\Users\$env:UserName\AppData\Local` is the same as `$env:LOCALAPPDATA`. So use native enviroment variables. :)

**Comment by pratyakshm** [comment]
> `C:\Users\$env:UserName\AppData\Local` is the same as `$env:LOCALAPPDATA`. So use native enviroment variables. :)

Hi! are you referring to the OneDrive uninstall code?

**Reply by farag2** [reply]
Strings 209—211, 394—396 in the CW-CLI.ps1 file.

**Reply by pratyakshm** [reply]
ah okay, I got it, I'll edit it later today. Thank you! :)
